src/vmm/src/linux/vstate.rs

```diff
@@ -1132,7 +1283,7 @@ impl Vcpu {
                 r => {
                     // TODO: Are we sure we want to finish running a vcpu upon
                     // receiving a vm exit that is not necessarily an error?
-                    error!("Unexpected exit reason on vcpu run: {:?}", r);
+                    error!("Unexpected exit reason on vcpu run: {:x?}", r);
```

Comment:
Should this be prefixed with 0x so the user knows it's in hex not decimal?

Reply:
I ultimately decided to drop the commit altogether, as I think it merely refers to a linux errno, without any addresses present.